api/apis.ts

```diff
@@ -1,3 +1,6 @@
+import querystring from 'querystring';
```

Comment:
tree-shaking should take care of ignoring unused modules.  These are native anyway, so I don't see any reason to do a complicated runtime import.